.github/ISSUE_TEMPLATE/doc-breaking-change.md

```diff
@@ -9,32 +9,51 @@ This issue template is for use in opening issues that document breaking changes.
 Text in brackets are placeholders; replace the text with the requested information and remove the brackets before submitting the issue. Also, remove this comment before submitting the issue.
 
 -->
-# [Brief description of the change]
+## [Change Title]
 
-## Version introduced
+[Brief description of the change]
 
-[Version in which the breaking change first occurred]
+### Version introduced
 
-## Old behavior
+[Version in which the breaking change first occurred (e.g., 3.0 for .NET Core 3.0)]
 
+### Change description
 
-## New behavior
+#### Old behavior
 
+#### New behavior
 
-## Reason for change
+#### Reason for change
 
 
-## Mitigations
+### Recommended action
 
-<!-- Do not modify anything below this line -->
+[ Suggested steps if user is affected go here:
+- Example of code changes to handle change
+- How to opt out of change ]
```

Comment:
Can you please remove the line "how to opt out of change", since opt out is not available in Core?